src/metakb/transform/civic.py

```diff
@@ -305,6 +305,7 @@ def _add_variant_therapeutic_response_studies(
                 r["variant_origin"], civic_gene
             )
 
+            _logger.debug("line 308")
```

Comment:
is this needed? along with the other `_logger.debug("line #")`?

Reply:
Oh thanks. This was what spurred me to try to do the log fixes in the first place.